dwave_networkx/generators/chimera.py

```diff
@@ -356,147 +357,119 @@ def __init__(self, m, n=None, t=4):
             The number of columns in the Chimera lattice.
         t : int, optional (default 4)
             The size of the shore within each Chimera tile.
-        """
-
-        self.args = m, m if n is None else n, t
-
-    def int(self, q):
-        """
-        Converts the chimera_index `q` into an linear_index
 
-        Parameters
-        ----------
-        q : tuple
-            The chimera_index node label    
+        See also
+        --------
+        :func:`.chimera_graph` for a description of the various conventions.
 
-        Returns
-        -------
-        r : int
-            The linear_index node label corresponding to q            
         """
+        self.args = m, m if n is None else n, 4 if t is None else t
 
+    def int(self, q):
+        """Deprecated alias for `chimera_to_linear`."""
+        msg = ('chimera_coordinates.int is deprecated and will be removed in '
+               'dwave-networkx 0.9.0, please use '
+               'chimera_coordinates.chimera_to_linear instead')
+        warnings.warn(msg, DeprecationWarning)
+        return self.chimera_to_linear(q)
+
+    def chimera_to_linear(self, q):
+        """Convert a 4-term Chimera coordinate to a linear index."""
         i, j, u, k = q
         m, n, t = self.args
         return ((n*i + j)*2 + u)*t + k
 
     def tuple(self, r):
-        """
-        Converts the linear_index `q` into an chimera_index
-
-        Parameters
-        ----------
-        r : int
-            The linear_index node label    
-
-        Returns
-        -------
-        q : tuple
-            The chimera_index node label corresponding to r
-        """
-
+        """Deprecated alias for `linear_to_chimera`."""
+        msg = ('chimera_coordinates.tuple is deprecated and will be removed in '
+               'dwave-networkx 0.9.0, please use '
+               'chimera_coordinates.linear_to_chimera instead')
+        warnings.warn(msg, DeprecationWarning)
+        return self.linear_to_chimera(r)
+
+    def linear_to_chimera(self, r):
+        """Convert a linear index to a 4-term Chimera coordinate."""
         m, n, t = self.args
         r, k = divmod(r, t)
         r, u = divmod(r, 2)
         i, j = divmod(r, n)
         return i, j, u, k
 
     def ints(self, qlist):
+        """Deprecated alias for `iter_chimera_to_linear`."""
+        msg = ('chimera_coordinates.ints is deprecated and will be removed in '
+               'dwave-networkx 0.9.0, please use '
+               'chimera_coordinates.iter_chimera_to_linear instead')
+        warnings.warn(msg, DeprecationWarning)
+        return self.iter_chimera_to_linear(qlist)
+
+    def iter_chimera_to_linear(self, qlist):
+        """Return an iterator converting 4-term Chimera coordinates to linear
```

Comment:
I think this needs to specify that the input is a list, something like, "converting a list of 4-term Chimera coordinates"

Reply:
You're right, in the pegasus ones it says "...converting a sequence of 4-term...", will fix.